omniduct/filesystems/local.py

```diff
@@ -94,6 +94,21 @@ def _mkdir(self, path, recursive):
                 raise
 
     # File opening
+    def _file_read_(self, path, size=-1, offset=0, binary=False):
```

Comment:
Using the omniduct specific File type is pretty sub-optimal at the moment, since it will read the entire file into memory on read operations. That could be improved, but even so, I think it's probably better to use the platform-optimised IO code in Python where we can. If this does lead to incompatibilities, I'd prefer to add hacks to the omniduct code than to lift everything into omniduct.